### ISSUE TITLE
Ambiguity with validateAttributeQuoteMarks

### ISSUE THREAD

**Issue by julienpa** [pugjs/pug-lint#104]
The doc contains the following example for a given rule `validateAttributeQuoteMarks: "'"`:
```pug
//- Invalid
input(type="text" name="name" value="value")

//- Valid
input(type='text' name='name' value='value')
```

But the following will be considered **valid** even if it could be considered **invalid** since it doesn't enforce the chosen style:
```pug
//- Valid
input(type=text name=name value=value)
```

**Comment by TimothyGu** [comment]
Well, the third snippet uses `text` as a variable. The rule is there to enforce string literal styles, but that third one doesn't use them as string literals.

**Comment by julienpa** [comment]
@TimothyGu yeah indeed, my issue was not accurate, sorry. The problem I am having on my current project is with number values for attribute, for example:
```pug
element(attr=40)
```
which outputs
```html
<element attr="40"></element>
```

...and I wish the linter could tell that `40` should have quotes around

**Comment by TimothyGu** [comment]
Well, the number works for the output too as you can see, so it's not really an issue for a linter to detect.

**Comment by julienpa** [comment]
The output also works if I use `"` sometimes and `'` on other occasions, yet there is a linter rule to ensure that I always use the same quote style.
My problem is that this rule doesn't apply to numbers, which **breaks** the feature of ensuring the **same quote style** for **_all_** attributes...
